sup3r/preprocessing/data_handling.py

```diff
@@ -408,6 +407,44 @@ def target(self, target):
         """Update target property"""
         self._target = target
 
+    @classmethod
+    def lats_are_descending(cls, lat_lon):
+        """Check if latitudes are in descending order (i.e. the target
+        coordinate is already at the bottom left corner)
+
+        Parameters
+        ----------
+        lat_lon : np.ndarray
+            Lat/Lon array with shape (n_lats, n_lons, 2)
+
+        Returns
+        -------
+        bool
+        """
+        return lat_lon[-1, 0, 0] < lat_lon[0, 0, 0]
+
+    @classmethod
+    def get_time_dim_name(cls, filepath):
+        """Get the name of the time dimension in the given file
+
+        Parameters
+        ----------
+        filepath : str
+            Path to the file
+
+        Returns
+        -------
+        time_key : str
+            Name of the time dimension in the given file
+        """
+        with xr.open_dataset(filepath) as handle:
+            valid_vars = set(handle.dims)
+            time_key = list({'time', 'Time'}.intersection(valid_vars))
+        if len(time_key) > 0:
+            return time_key[0]
+        else:
+            return 'time'
```

Comment:
why is this a class method instead of static? Can you make a utility function for this since the same functionality is needed in post processing above? if not, move to the nc specific classes? Can you update the docstring to clarify that this is just for .nc?